File: test/fixtures/importmap-laravel-eager/routes/importmap.php
<?php

use Tonysm\ImportmapLaravel\Facades\Importmap;

Importmap::pinAllFrom("resources/js", to: "js/");

Importmap::pin("@hotwired/stimulus-loading", to: "/vendor/stimulus-laravel/stimulus-loading.js");
Importmap::pin("@hotwired/stimulus", to: "/js/vendor/@hotwired--stimulus.js"); // @hotwired/stimulus@3.2.2 downloaded from https://ga.jspm.io/npm:@hotwired/stimulus@3.2.2/dist/stimulus.js

File: test/fixtures/importmap-rails-inheritance/config/importmap.rb
# Pin npm packages by running ./bin/importmap

pin "application"

pin "application", preload: true
pin "@hotwired/stimulus", to: "stimulus.min.js"
pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"

pin_all_from "app/javascript/controllers", under: "controllers"
